File: app/src/main/java/edgar/yodgorbek/sportnews/model/TalkSports.java
package edgar.yodgorbek.sportnews.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class TalkSports {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("totalResults")
    @Expose
    private Integer totalResults;
    @SerializedName("articles")
    @Expose
    private List<Article> articles = null;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public Integer getTotalResults() {
        return totalResults;
    }

    public void setTotalResults(Integer totalResults) {
        this.totalResults = totalResults;
    }

    public List<Article> getSportArticles() {
        return articles;
    }

    public void setSportArticles(List<Article> articles) {
        this.articles = articles;
    }

}

File: app/src/main/java/edgar/yodgorbek/yangiliklar/model/TalkSports.kt
package edgar.yodgorbek.yangiliklar.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class TalkSports {

    @SerializedName("status")
    @Expose
    var status: String? = null
    @SerializedName("totalResults")
    @Expose
    var totalResults: Int? = null
    @SerializedName("articles")
    @Expose
    var sportArticles: List<Article>? = null

}


